scripts/lib/cluster.sh

```diff
@@ -28,9 +32,14 @@ function up-test-cluster() {
         AWS_K8S_TESTER_EKS_ADD_ON_NLB_HELLO_WORLD_ENABLE=true \
         AWS_K8S_TESTER_EKS_ADD_ON_ALB_2048_ENABLE=true \
         AWS_K8S_TESTER_EKS_KUBECTL_PATH=$KUBECTL_PATH \
-    $TESTER_PATH eks create config --path $CLUSTER_CONFIG 1>&2
-    echo -n "Creating cluster $CLUSTER_NAME (this may take ~20 mins. details: tail -f $CLUSTER_MANAGE_LOG_PATH)... "
-    $TESTER_PATH eks create cluster --path $CLUSTER_CONFIG 2>>$CLUSTER_MANAGE_LOG_PATH 1>&2 ||
-        ( echo "failed. Check $CLUSTER_MANAGE_LOG_PATH." && exit 1 )
-    echo "ok."
+        $TESTER_PATH eks create config --path $CLUSTER_CONFIG 1>&2
+
+    if [[ -n "${CIRCLE_JOB:-}" ]]; then
+        $TESTER_PATH eks create cluster --path $CLUSTER_CONFIG || (echo "failed!" && exit 1)
+    else
+        echo -n "Creating cluster $CLUSTER_NAME (this may take ~20 mins. details: tail -f $CLUSTER_MANAGE_LOG_PATH)... "
+        $TESTER_PATH eks create cluster --path $CLUSTER_CONFIG 2>> $CLUSTER_MANAGE_LOG_PATH 1>&2 ||
+            (echo "failed. Check $CLUSTER_MANAGE_LOG_PATH." && exit 1)
+        echo "ok."
+    fi
```

Comment:
I'm confused why you're changing the above depending on whether it's running in CI or not?

Reply:
It's just changing the logging output. By default, CircleCI will terminate any test after 10 minutes fi there is no console output. That can be changed by setting `no_output_timeout` for the job, but when you have to wait 40 minutes it's more interesting to see what is going on.

Reply:
ack, makes sense. ++